src/frontend/src/components/AboutFooter/AboutFooter.jsx

```diff
@@ -51,6 +52,9 @@ const AboutFooter = () => {
   const classes = useStyles();
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.up('md'));
+  const { contributor, loading, error } = useTelescopeContributor();
```

Comment:
I doubt that we care about `loading`, and can use the other 2 to do what we want.